pkg/build/gobuild.go

```diff
@@ -784,7 +817,7 @@ func (g *gobuild) buildOne(ctx context.Context, refStr string, base v1.Image, pl
 		cfg.Config.Entrypoint = []string{`C:\ko-app\` + appFilename(ref.Path())}
 		updatePath(cfg, `C:\ko-app`)
 		cfg.Config.Env = append(cfg.Config.Env, `KO_DATA_PATH=C:\var\run\ko`)
-	} else {
+	} else if platform.String() != "wasm/wasi" {
```

Comment:
If we keep doing this comparison, I wonder if it warrants having a helper method `isWasi(platform)` (and `isWindows(platform)`) to make this easier to read.

Then we can also avoid the string comparison without bloating the callsite:

```
func isWasi(p v1.Platform) bool {
  return p.OS == "wasm" && p.Architecture == "wasi"
}
```

Reply:
Yes, this is a good idea. The constant comparison with platform string was kinda hacky anyway.